story.txt

```diff
@@ -1,7 +1,6 @@
 
 You would never guess what can happen when you jump into a seemingly shallow puddle at night time!
-It turns out that it is not a puddle rather than a giant hole which brings you to a new world on the other side of the light water.
-QNow you are left stunning and don't know what to do.
+It turns out that it is not a puddle rather than a giant hole which brings you to a new world on the other side of the light water.QNow you are left stunning and don't know what to do.
```

Comment:
Looks like you accidentally added a `Q` in this line. Please remove this and commit again : )